thanos/plugindefinition.yaml

```
@@ -9,8 +9,8 @@ spec:
     description: thanos
     helmChart:
         name: thanos
-        repository: "oci://keppel.eu-de-1.cloud.sap/greenhouse-extensions"
-        version: 0.0.8
+        repository: "oci://ghcr.io/cloudoperators/greenhouse-extensions/charts "
```

There is a trailing blank space. Is this intended? 
```suggestion
        repository: "oci://ghcr.io/cloudoperators/greenhouse-extensions/charts"
```